backend/django/core/templates/projects/create/create_wizard_labels.html

```diff
@@ -104,25 +112,29 @@ <h3>Instructions</h3>
           var csv = event.target.result;
           var data = $.csv.toArrays(csv);
           if(data[0][0] === 'Label' && data[0][1] === 'Description') {
-            data.forEach((label, index) => {
-              if(index !== 0) {
-                if($('.formset_row_label').eq(index - 1).length > 0) {
-                  $('.formset_row_label').eq(index - 1).css('display', '');
-                  $('.formset_row_label').eq(index - 1).find('input[type=text]').eq(0).val(label[0]);
-                  if(label[1].length > 0) {
-                    $('.formset_row_label').eq(index - 1).find('textarea').eq(0).val(label[1]);
-                  } else {
-                    $('.formset_row_label').eq(index - 1).find('textarea').eq(0).val('');
-                  }
-                } else {
-                  $(':contains("add label")').click();
```

Comment:
So when we have a bunch of labels add by csv, the "add label" button seems to go away at the bottom. However I think this is also true of the current version in npsas_main. Do we know why this happens?

Reply:
Swapping out setting the HTML with `prepend()` from JQuery did the trick. I think the add button is part of the table, so when the HTML was being overwritten that was also removed.

Reply:
Cool! So I'm breaking things when I do the following:

1. Make a project and uploade a huge label file
2. delete the first label
3. hit continue

It seems to clear the label, but then gives an error that it's empty?
![Screen Shot 2022-05-25 at 10 46 48 AM](https://user-images.githubusercontent.com/10951569/170292361-4370a663-fa93-43e0-99e7-423f36a49da1.png)

Also the add label button vanishes again lol.
![Screen Shot 2022-05-25 at 10 47 40 AM](https://user-images.githubusercontent.com/10951569/170292547-05a6f9f9-6572-4b09-9a7a-ad408043a657.png)

Reply:
I'll note that before I did 2 and 3, the add label button was indeed there

Reply:
It was some weird Django requirement that you can't delete labels 1 and 2 but have to use a hidden input and turn them off. Only applied to those two. I added a fix to account for that so hopefully it works smoothly now.

Reply:
So it looks like currently if you delete one of the first or second ones then both are removed.

The behavior to replicate:

1. load in the cip_codes.csv file
2. delete either the first or second label
3. Go to the next page
4. Go back to the previous page
5. wait for it to finish loading (there's an odd in-between page with a delete column too. hmmm)